weather_mv/loader_pipeline/util_test.py

```diff
@@ -178,5 +185,99 @@ def test_make_attrs_ee_compatible_a2(self):
 
 
 class ToJsonSerializableTypeTests(unittest.TestCase):
-    # TODO(#106): Write tests...
-    pass
+
+    def test_to_json_serializable_type_none(self):
+        attrs = None
+
+        expected = None
+
+        actual = to_json_serializable_type(attrs)
+
+        self.assertEqual(actual, expected)
+        self.assertEqual(type(actual), type(expected))
```

Comment:
I don't think we need to include a type check; equality should be sufficient

Reply:
Updated in https://github.com/google/weather-tools/pull/327/commits/d9cf97add436e462d01fb43fe2d32ac7c0511d17.